<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" type="image/png" sizes="96x96" href="favicon.png">

    <title>AACBB - Workshop on Accelerator Architecture for Computational Biology and Bioinformatics</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom fonts for this template -->
    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

    <!-- Custom styles for this template -->
    <link href="css/agency.css" rel="stylesheet">

  </head>

  <body id="page-top">

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
      <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">Back to top</a>
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          Menu
          <i class="fa fa-bars"></i>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#about">About</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#cfp">Call for Participation</a>
            </li>

            <li class="nav-item">
              <!-- <a class="nav-link js-scroll-trigger" href="#invited-speakers">Keynote Speakers</a> -->
			  <a class="nav-link js-scroll-trigger" href="#schedule">Schedule</a>
            </li>
              <li class="nav-item">
                  <a class="nav-link js-scroll-trigger" href="#program-committee">Program Committee</a>
              </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="AACBB-2019.html" target="_blank">AACBB'19</a>
            </li>
			
		<!-- 	<li class="nav-item">
              <a class="nav-link" href="AACBB-2018.html" target="_blank">AACBB'18</a>
            </li> -->

          </ul>
        </div>
      </div>
    </nav>

    <!-- Header -->
    <header class="masthead">
        <div class="container">
            <div class="intro-text">
                <div class="intro-lead-in">The 3<sup>rd</sup> HPCA Workshop on</div>
                <div class="intro-heading">Accelerator Architecture in <br> Computational Biology and Bioinformatics</div>

            </div>
            <h3 class="heading-subtitle">February 23rd, 2020</h3>
            <h3 class="heading-subtitle">San Diego, CA, USA</h3>
        </div>
    </header>


    <section id="about">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">About</h2>
            <p class="about-text">Over the last decade, the advent of high-throughput sequencing techniques brought an exponential growth in biosequence database sizes. 
			With increased throughput demand and popularity of computational biology tools, reducing time-to-solution during computational analysis has become a significant challenge in the path to scientific discovery.<br>
			
                <br>Conventional computer architecture has been proven to be inefficient for computational biology and bioinformatics tasks. 
				For example, 3<sup>rd</sup> generation de-novo assembly of a human genome may take hundreds of hours on a high performance computer. 
				Hence, computational biology and bioinformatics increasingly rely on hardware accelerators.<br>
				
                <br>In a typical application, the dominant portion of the runtime is spent in a small number of computational kernels, making it an excellent target for hardware acceleration. 
				The combination of increasingly large datasets and high performance computing requirements makes computational biology a prime candidate to benefit from accelerator architecture research. 
				Potential directions include 3D integration, near-data processing, automata processing and reconfigurable architectures.<br>
				
				<br>This workshop will focus on the architecture and design of hardware accelerators for computational biology and bioinformatics problems. We plan to present and discuss a variety of acceleration techniques,
				accelerator architectures and their implications on the development of computational biology.<br>
            </p>
          </div>
        </div>
      </div>
    </section>


    <section id="cfp">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 ">
                    <h2 class="section-heading text-center">Call For Participation</h2>
                    <h5>
                        This workshop focuses on architecture and design of hardware and software accelerators for computational biology and bioinformatics problems.
                        <br><br>Topics of interest include, but are not limited to the following:
                    </h5>
                    <ul>
                        <li>Hardware and software algorithms/applications in the fields of computational biology, such as (but not limited to):</li>
						<ul style="padding-left:20px">
							<li>Bioinformatics</li>
							<li>Genomics</li>
							<li>Proteomics</li>
							<li>Protein structure prediction</li>
						</ul>
                        <li>Bioinformatics and computational biology accelerator architecture and design based on (but not limited to):</li>
						<ul style="padding-left:20px">
							<li>3D memory-logic stack</li>
							<li>Automata processing</li>
							<li>Near-data (in-memory) processing</li>
							<li>FPGAs and reconfigurable</li>
						</ul>
                        <li>Emerging memory technologies and their impact on bioinformatics and computational biology</li>
						<li>Impact of bioinformatics and biology applications on computer architecture research</li>
						<li>Bioinformatics and computational biology-inspired hardware/software trade-offs</li>
                    </ul>
					
					<h2> HPCA information</h2>
                    <h6><br><a href = "https://www.hpca-conf.org/2020/registration/">Registration link</a></h6>
					<h6><br><a href = "https://www.hpca-conf.org/2020/venue-hotel/">Venue and hotel information</a></h6>
					
                   <!--  <h5 class="padding-bottom-10">Interested authors are encouraged to submit papers (2-5 pages) through 
                        <a href="https://easychair.org/conferences/?conf=aacbb2020">EasyChair</a>
					<ul>
                        <li><h6>Submitting and presenting a paper in the workshop does not preclude publication in other venues.</h6></li>
                    </ul>
                    </h5> -->
                    <!-- <h6>For a PDF version of the CFP click here: <a href="CFP/Workshop%20on%20AACBB%20-%202019%20-%20extended%20CFP.pdf" target="_blank">Download CFP</a></h6>
                    <p>All papers will be evaluated based on their relation to the subject, novelty, fundamental insights, experimental evaluation, and potential for long-term impact.</p> -->
<!--                     <p>The full version should be a PDF file in HPCA26 submission format. For formatting instructions please refer to:
                        <a href="https://www.hpca-conf.org/2020/hpca-2020-guidelines-for-submissions/">HPCA Guidelines for Submission</a>
                    </p>

                     <h2>Important Dates</h2>
                    <h5><u>Submission deadline</u>: <del>Dec 15th, 2019 </del><ins><b><font size="6">Dec 30th, 2019</font></b></ins></h5>
					 -->

                    <!--<h5><u>Notification by</u>: Dec 28th, 2018.</h5>
                    <h3 class="margin-top-30">Important Notes</h3>
                    <ul>
                        <li><h6>Submitting and presenting a paper in the workshop does not preclude publication in other venues (there are no proceedings to this workshop).</h6></li>
                    </ul>


                    <h4>A tentative workshop schedule can be found <a href="#schedule">below</a></h4> -->
                   

                </div>
            </div>
        </div>
    </section>

    <!-- About -->
	
	<section id="schedule">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Schedule</h2>
          </div>
        </div>
        <div class="row padding-bottom-20">
            <div class="col-lg-1"></div>
            <div class="col-lg-10">
                <table id="scheulde-table" class="table-striped table-responsive table-hover">
                    <tr>
                        <th style="width:120px"></th>
                        <th></th>
                    </tr>
                    <tr>
                        <td>7:00 - 8:20</td>
                        <td>Breakfast</td>
                    </tr>

                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>8:20 - 8:30</td>
                        <td>Opening Remarks</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td><strong>8:30-9:30</strong></td>
                        <td><strong>James Larus, EPFL </strong><br>
                                <i>"Protein Clustering: Parallelizing an Expensive Irregular Computation"</i>
                            <!-- &nbsp;(<a target="_blank" href="./slides/2019/impact-aacbb19-2019_02_16.pptx">slides</a>) -->
                        </td>
                    </tr>
                    <tr>
                        <td>9:30 - 10:00</td>
                        <td>Farzaneh Zokaee and Lei Jiang<br>
                            <!-- <br><a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bby017/4958758?redirectedFrom=fulltext" target="_blank"> -->
                                <i>"FindeR: Accelerating FM-Index-based Exact Pattern Matching in Genomic Sequences through ReRAM technology"</i>
                            <!-- &nbsp;(<a target="_blank" href="./slides/2019/HPCA_AACBB_2019_Nanopore_talk.pdf">slides</a>) -->
                        </td>
                    </tr>

                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>10:00 - 10:30</td>
                        <td>Coffee break</td>
                    <tr><td><br></td><td><br></td></tr>

                        <td><strong>10:30 - 11:30</strong></td>
                        <td><strong>Daniel Jacobson, Oak Ridge National Laboratory</strong>
                            <br><i>"Exascale Biology: From Genome to Climate with a Few Steps Along the Way"</i>
                            </td>
                    </tr>


                    <!-- Paper session 2 (3 regular papers) -->
                    <tr>
                        <td>11:30 - 12:00</td>
                        <td>Fan Chen, Linghao Song, Hai Li and Yiran Chen
                            <br><i>"PARC: A Processing-in-CAM Architecture for Genomic Long Read Pairwise Alignment using ReRAM"</i></td>

                    </tr>
					<tr><td><br></td><td><br></td></tr>
                    
                    <tr>
                        <td>12:00 - 13:00</td>
                        <td>Lunch</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td><strong>13:00 - 14:00</strong></td>
                        <td><strong>Benedict Paten, UC Santa Cruz Genomics Institute</strong>
                            <br><i>"Building the Human Pangenome"</i></strong></td>
                    </tr>
					
					<tr>
                        <td>14:00 - 14:30</td>
                        <td>Wenqin Huangfu, Xueqi Li, Shuangchen Li, Xing Hu, Peng Gu and Yuan Xie
                            <br><i>"MEDAL: Scalable DIMM based Near Data Processing Accelerator for DNA Seeding Algorithm"</i></td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td>14:30 - 15:00</td>
                        <td>Coffee break</td>
                    </tr>
					
                    <tr><td><br></td><td><br></td></tr>
					
                    <tr>
                        <td>15:00 - 15:30</td>
                        <td>Saransh Gupta, Mohsen Imani, Behnam Khaleghi, Venkatesh Kumar and Tajana Rosing
                            <br>
                            <i>"RAPID: A ReRAM Processing in-Memory Architecture for DNA Sequence Alignment"</i>
                        </td>
                    </tr>
                    <tr>
                        <td>15:30 - 16:00</td>
                        <td>Zheming Jin, Kazutomo Yoshii and Hal Finkel
                            <br><i>"Optimizing the Performance of the Needleman-Wunsch Benchmark using SYCL"</i></a>
                        </td>
                    </tr>
					
					<tr>
                        <td>16:00 - 16:30</td>
                        <td>Roman Snytsar and Yatish Turakhia
                            <br><i>"Parallel acceleration of ntHash"</i></a>
                        </td>
                    </tr>

                    <tr>
                        <td>16:30 - 17:00</td>
						<td>Chih-Han Yang, Jhih-Wun Zeng, Cheng-Yueh Liu and Shih-Hao Hung
                            <br><i>"Accelerating Variant Calling via Performance Analysis and Optimization of DeepVariant"</i></a>
                        </td>
                    </tr>

                    <tr>
                        <td>17:00 - 17:05</td>
                        <td>Closing remarks</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>


                </table>
            </div>

        </div>
	

		<div class="container">
            <div class="row">
                <div class="col-sm-2"></div>
                <div class="col-lg-8 ">
                    <h2 class="section-heading text-center">Keynote Speakers</h2>
<!--                     <ul id="program-committee-list">
                        <li><a href="https://www.soe.ucsc.edu/people/benedict">Benedict Paten, UC Santa Cruz Genomics Institute</a></li>
                         <li><a href="https://people.epfl.ch/james.larus?lang=en">James Larus, EPFL</a></li>
                         <li><a href="https://www.deshawresearch.com/people_hdwdes_gagliardo.html">Joe Gagliardo, D. E. Shaw Research</a></li>
                     
					</ul> -->
                </div>
            </div>
        </div>

	
		<div class="row">
          <div class="col-lg-12">
                <!-- <h2 class="section-heading text-center">Talks</h2> -->
                <ul class="timeline">

                    <li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/2020/James_Larus.jpg" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://people.epfl.ch/james.larus?lang=en">James Larus, EPFL</a></h4>
                                <h3 class="keynote-subheading">"Protein Clustering: Parallelizing an Expensive Irregular Computation"</h3>
                                <!--<h5 class="subheading">8:20-9:20</h5> !-->
                            </div>
                            <div class="timeline-body">
                                <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
                                    James Larus is Professor and Dean of the School of Computer and Communication Sciences (IC) at EPFL (École Polytechnique Fédérale de Lausanne). 
									Prior to joining IC in October 2013, Larus was a researcher, manager, and director in Microsoft Research for over 16 years and an assistant and associate professor in the Computer Sciences Department 
									at the University of Wisconsin, Madison. Larus has been an active contributor to numerous communities. 
									He published over 100 papers (with 9 best and most influential paper awards), received over 40 US patents. 
									Larus received a National Science Foundation Young Investigator award in 1993 and became an ACM Fellow in 2006. 
									Larus received his MS and PhD in Computer Science from the University of California, Berkeley in 1989, and an AB in Applied Mathematics from Harvard in 1980.
                                </p>
                            </div>
                        </div>
                    </li>
					
					
					<li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/2020/Benedict_Paten.png" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://www.soe.ucsc.edu/people/benedict">Benedict Paten, UC Santa Cruz Genomics Institute</a></h4>
                                <h3 class="keynote-subheading">"Building the Human Pangenome"</h3>
                                <!--<h5 class="subheading">8:20-9:20</h5> !-->
                            </div>
                            <div class="timeline-body">
                                <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
                                    Dr. Benedict Paten is an assistant professor in the department of Biomolecular Engineering at the University of California Santa Cruz (UCSC) and 
									an associate director of the UC Santa Cruz Genomics Institute. He directs the Computational Genomics Lab at UCSC, which 
									is broadly focused on computational genomics, creating algorithms, software and services addressing biomolecular challenges. 
									He has a PhD from the University of Cambridge and the European Molecular Biology Laboratory in computational biology.
                                </p>
                            </div>
                        </div>
                    </li>
					
					
					
                    <li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/2020/Daniel_Jacobson.png" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://www.ornl.gov/staff-profile/daniel-jacobson">Daniel Jacobson, Oak Ridge National Laboratory</a></h4>
                                <h3 class="keynote-subheading">"Exascale Biology:  From Genome to Climate with a Few Steps Along the Way"</h3>
                                <!--<h5 class="subheading">8:20-9:20</h5> !-->
                            </div>
                            <div class="timeline-body">
                                <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
									Dr. Jacobson is the Chief Scientist for Computational Systems Biology at the Oak Ridge National Laboratory. 
									His lab was the first group to break the Exascale barrier and is happy to have done so for a biology project. 
									At present, this (2.41 Exaops) calculation is the fastest scientific calculation ever done anywhere in the world. This project led to his team being awarded the 2018 Gordon Bell 
									Prize (the first ever for Systems Biology). <br>Dr. Jacobson's career as a computational systems biologist has included leadership roles in academic, corporate, NGO and national lab settings. 
									His lab focuses on the development and subsequent application of mathematical, statistical and computational methods to biological datasets in order to yield new insights into 
									complex biological systems.  His lab’s approaches include the use of Network Theory and Topology Discovery/Clustering, Wavelet Theory, and explainable-AI, together with traditional 
									and more advanced supercomputing architectures.  Areas of statistics of particular interest to his lab include the use of both frequentist (parametric and non-parametric) and 
									Bayesian methods as well as the development of new methods for Genome-Wide Epistasis Studies (GWES). 
									These mathematical and statistical methods are applied to various population and (meta)multiomics data sets (Genomics, Phylogenomics, Transcriptomics, Proteomics, Metabolomics, 
									Microbiomics, Viriomics, Phytobiomics, Chemiomics, etc.) individually as well as in combination in an attempt to better understand the functional relationships as well as biosynthesis, 
									signaling, transcriptional, translational, degradation and kinetic regulatory networks at play in biological organisms and communities. 
									His group takes a broad view of biological complexity and evolution that stretches from viruses to microbes to plants to humans. 
									ORNL is home to some of the world’s largest supercomputers and thus his lab uses petascale and exascale computing to analyze and model complex biological systems.
                                </p>
                            </div>
                        </div>
                    </li>
					
					
					
                </ul>
            </div>
        </div>
		

		
    </section>
	
	
	

    <section id="program-committee">
        <div class="container">
            <div class="row">
                <div class="col-sm-2"></div>
                <div class="col-lg-8 ">
                    <h2 class="section-heading text-center">Program Committee</h2>
                    <ul id="program-committee-list">
                        <li><a href="http://www.eecs.wsu.edu/~ananth/">Ananth Kalyanaraman, WSU</a></li>
                        <li><a href="http://www.cs.bilkent.edu.tr/~calkan/">Can Alkan, Bilkent University</a></li>
                        <li><a href="http://www.cs.rochester.edu/~ipek/Home.html">Engin Ipek, University of Rochester</a></li>
                        <li><a href="http://vast.cs.ucla.edu/people/faculty/jason-cong">Jason Cong, UCLA</a></li>
                        <li><a href="http://lph.ece.utexas.edu/merez/MattanErez/Home">Mattan Erez, UT Austin</a></li>
                        <li><a href="http://www.ece.virginia.edu/faculty/stan.html">Mircea Stan, UVA</a></li>
                        <li><a href="https://people.inf.ethz.ch/omutlu/">Onur Mutlu, ETH/CMU</a></li>
                        <li><a href="http://webee.technion.ac.il/~ran/">Ran Ginosar, Technion</a></li>
						<li><a href="https://asic2.group/team/current-members/?stl=2066">Ronnie Ronen, Technion</a></li>
                        <li><a href="http://www.ece.ucsb.edu/~yuanxie/">Yuan Xie, UCSB</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </section>

	
	

    <!-- Organizers -->
    <section class="bg-light" id="organizers">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Workshop Organizers</h2>
            <h3 class="padding-bottom-15 text-muted">From Technion, Israel Institute of Technology</h3>
          </div>
        </div>
        <div class="row">
            <!--<div class="col-sm-1"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Leonid.jpg" alt="">
              <h4>Leonid Yavits</h4>
              <p class="oranizer-email">leonid.yavits@nububbles.com</p>
                <p class="text-left"><strong>Short bio:</strong> Leonid received his MSc and PhD in Electrical Engineering from the Technion. After graduating, he co-founded VisionTech where he co-designed a single chip MPEG2 codec. Following VisionTech’s acquisition by Broadcom, he co-founded Horizon Semiconductors where he co-designed a Set Top Box on chip for cable and satellite TV. <br>
                    Leonid is a postdoc fellow in Electrical Engineering in the Technion. He co-authored a number of patents and research papers on SoC and ASIC. His research interests include non von Neumann computer architectures and processing in memory</p>
            </div>
          </div>
            <!--<div class="col-sm-2"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Naor.jpeg" alt="">
              <h4>Naor Granik</h4>
              <p class="oranizer-email">naorgranik@gmail.com</p>
                <p class="text-left"><strong>Short bio:</strong>
                    Naor received his BSc from the faculty of Electrical Engineering, and MSc from the faculty of Biomedical Engineering, Technion, Israel in 2017 and 2019, respectively. He is now a PhD candidate in the faculty of Applied Mathematics under the supervision of Prof. Roee Amit. <br>
					Naor's research interests are near-data processing, deep-learning and biological computing.
                </p>

            </div>
          </div>
        </div>
		
		<div class="row">
          <div class="col-lg-12 text-center">
            <h3 class="padding-bottom-15 text-muted">From Stanford</h3>
          </div>
        </div>
		
		
		<div class="row">
            <!--<div class="col-sm-1"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Yatish.jpg" alt="">
              <h4>Yatish Turakhia</h4>
              <p class="oranizer-email">yatisht@stanford.edu</p>
                <p class="text-left"><strong>Short bio:</strong> Yatish is a final year PhD student in Electrical Engineering at Stanford University, where he is co-advised by Prof. Bill Dally and Prof. Gill Bejerano.
				He is interested in the hardware-acceleration of genome sequence alignment and its application to biomedicine and comparative genomics. 
				His work has won the best paper award at ASPLOS 2018 and IEEE Micro Top picks 2018 award. He is also a recipient of the NVIDIA graduate fellowship. 
				He has served as a reviewer in a number of top-tier computer architecture journals, including Transactions on Computers, Transactions on Architecture and Code Optimization (TACO) and ComputerArchitecture Letters (CAL)</p>
            </div>
          </div>
            <!--<div class="col-sm-2"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Sneha.jpg" alt="">
              <h4>Sneha Goenka</h4>
              <p class="oranizer-email">gsneha@stanford.edu</p>
                <p class="text-left"><strong>Short bio:</strong>
                    Sneha is a third year Ph.D. Candidate at Stanford University where she is advised by Prof. Mark Horowitz and Prof. Gill Bejerano. Previously, she received her B. Tech. and M. Tech. 
					in Electrical Engineering at the Indian Institute of Technology, Bombay. She is interested in developing co-processors for genomic applications.
                </p>

            </div>
          </div>
        </div>
		
        <!--<div class="row">
          <div class="col-lg-8 mx-auto text-center">
            <p class="large text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde.</p>
          </div>
        </div>-->
      </div>
    </section>

    <!-- Contact -->
    <!--
    <section id="contact">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Contact Us</h2>
            <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
          </div>
        </div>
        <div class="row">
          <div class="col-lg-12">
            <form id="contactForm" name="sentMessage" novalidate>
              <div class="row">
                <div class="col-md-6">
                  <div class="form-group">
                    <input class="form-control" id="name" type="text" placeholder="Your Name *" required data-validation-required-message="Please enter your name.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="email" type="email" placeholder="Your Email *" required data-validation-required-message="Please enter your email address.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="phone" type="tel" placeholder="Your Phone *" required data-validation-required-message="Please enter your phone number.">
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="col-md-6">
                  <div class="form-group">
                    <textarea class="form-control" id="message" placeholder="Your Message *" required data-validation-required-message="Please enter a message."></textarea>
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="clearfix"></div>
                <div class="col-lg-12 text-center">
                  <div id="success"></div>
                  <button id="sendMessageButton" class="btn btn-xl" type="submit">Send Message</button>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    </section>
    -->

    <!-- Footer -->
    <footer>
      <div class="container">
        <div class="row">
          <div class="col-md-4">
            <span class="copyright">Copyright &copy; AACBB 2020</span>
          </div>

        </div>
      </div>
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/popper/popper.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

    <!-- Contact form JavaScript -->
    <script src="js/jqBootstrapValidation.js"></script>
    <script src="js/contact_me.js"></script>

    <!-- Custom scripts for this template -->
    <script src="js/agency.js"></script>

  </body>

</html>
