<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" type="image/png" sizes="96x96" href="favicon.png">

    <title>AACBB - Workshop on Accelerator Architecture for Computational Biology and Bioinformatics</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom fonts for this template -->
    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

    <!-- Custom styles for this template -->
    <link href="css/agency.css" rel="stylesheet">

  </head>

  <body id="page-top">

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
      <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">Back to top</a>
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          Menu
          <i class="fa fa-bars"></i>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#about">About</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#cfp">Call for Participation</a>
            </li>

            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#schedule">Schedule</a>
            </li>
              <li class="nav-item">
                  <a class="nav-link js-scroll-trigger" href="#program-committee">Program Committee</a>
              </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="AACBB-2018.html" target="_blank">1<sup>st</sup> AACBB</a>
            </li>

          </ul>
        </div>
      </div>
    </nav>

    <!-- Header -->
    <header class="masthead">
        <div class="container">
            <div class="intro-text">
                <div class="intro-lead-in">The 2<sup>nd</sup> HPCA Workshop on</div>
                <div class="intro-heading">Accelerator Architecture in <br> Computational Biology and Bioinformatics</div>

            </div>
            <h3 class="heading-subtitle">February 16th, 2019</h3>
            <h3 class="heading-subtitle">Washington D.C., USA</h3>
        </div>
    </header>


    <section id="about">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">About</h2>
            <p class="about-text">Over the last decade, the advent of high-throughput sequencing techniques brought an exponential growth in biosequence database sizes. With increased throughput demand and popularity of computational biology tools, reducing time-to-solution during computational analysis has become a significant challenge in the path to scientific discovery.<br>
                <br>Conventional computer architecture is proven to be inefficient for computational biology and bioinformatics tasks. For example, aligning even several hundred DNA or protein sequences using progressive multiple alignment tools consumes several CPU hours on high performance computer. Hence, computational biology and bioinformatics rely on hardware accelerators to allow processing to keep up with the increasing amount of data generated from biology applications.<br>
                <br>In a typical application, dominant portion of the runtime is spent in a small number of computational kernels, making it an excellent target for hardware acceleration. The combination of increasingly large datasets and high performance computing requirements make computational biology prime candidate to benefit from accelerator architecture research. Potential directions include 3D integration, near-data processing, automata processing, associative processing and reconfigurable architectures.<br>
            </p>
          </div>
        </div>
      </div>
    </section>


    <section id="cfp">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 ">
                    <h2 class="section-heading text-center">Call For Participation</h2>
                    <h5>
                        This workshop focuses on architecture and design of hardware and software accelerators for computational biology and bioinformatics problems.
                        <br><br>Topics of interest cover the following:
                    </h5>
                    <ul>
                        <li>Impact of bioinformatics and biology applications on computer architecture research</li>
                        <li>Bioinformatics and computational biology accelerator architecture and design</li>
                        <li>3D memory-logic stack based accelerators</li>
                        <li>Automata processing in bioinformatics and computational biology applications</li>
                        <li>Associative processing in bioinformatics and computational biology applications</li>
                        <li>Near-data (in-memory) acceleration of bioinformatics and computational biology applications</li>
                        <li>Emerging memory technologies and their impact on bioinformatics and computational biology</li>
                        <li>Embedded and reconfigurable architectures</li>
                        <li>Field programmable logic based accelerators</li>
                        <li>Bioinformatics and computational biology-inspired hardware/software trade-offs</li>
                        <li>Software acceleration of computational biology and bioinformatics</li>
                    </ul>

                    <!--<h2>Submission Guidelines</h2>
                    <h5 class="padding-bottom-10">Interested authors are encouraged to submit papers (2-5 pages) through EasyChair:
                        <a href="https://easychair.org/conferences/?conf=aacbb2019">https://easychair.org/conferences/?conf=aacbb2019</a>
                    </h5>
                    <h6>For a PDF version of the CFP click here: <a href="CFP/Workshop%20on%20AACBB%20-%202019%20-%20extended%20CFP.pdf" target="_blank">Download CFP</a></h6>
                    <p>All papers will be evaluated based on their relation to the subject, novelty, fundamental insights, experimental evaluation, and potential for long-term impact.</p>
                    <p>The full version should be a PDF file in HPCA25 submission format. For formatting instructions please refer to:
                        <a href="http://hpca2019.seas.gwu.edu/guidelines_for_submission.html">HPCA Guidelines for Submission</a>
                    </p>

                    <h2>Important Dates</h2>
                    <h5><u>Submission deadline</u>: <strike>Dec 15th</strike> Dec 25th, 2018, EoD AoE.</h5>
                    <h5><u>Notification by</u>: Dec 28th, 2018.</h5>

                    <h3 class="margin-top-30">Important Notes</h3>
                    <ul>
                        <li><h6>Submitting and presenting a paper in the workshop does not preclude publication in other venues (there are no proceedings to this workshop).</h6></li>
                        <li><h6>We will have a poster session. Some papers might be accepted as <i>poster papers</i>.</h6></li>
                        <li><h6>We plan to have a lightning round, where the authors of a poster paper are given an opportunity to present their work to all attendants in a “lightning” 2 min presentation.</h6></li>
                    </ul>

                    <h4>A tentative workshop schedule can be found <a href="#schedule">below</a></h4>
                    -->

                </div>
            </div>
        </div>
    </section>

    <!-- About -->
    <section id="schedule">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Schedule</h2>
          </div>
        </div>
        <div class="row padding-bottom-20">
            <div class="col-lg-1"></div>
            <div class="col-lg-10">
                <table id="scheulde-table" class="table-striped table-responsive table-hover">
                    <tr>
                        <th style="width:120px"></th>
                        <th></th>
                    </tr>
                    <tr>
                        <td>8:40 - 8:45</td>
                        <td>Opening Remarks</td>
                    </tr>
                    <tr>
                        <td><strong>8:45 - 9:45</strong></td>
                        <td><strong>Keynote 1: William Dally (Stanford & NVIDIA)</strong>
                            <br>
                            <i>“Accelerating Genome Analysis: A Primer on an Ongoing Journey”</i></strong>&nbsp;
                            <!--(<a href="./slides/2018/onur-AcceleratingGenomeAnalysis-AACBB_Keynote-February-24-2018-FINAL.pptx">slides</a>-->
                        </td>

                    </tr>
                    <!-- Paper session 1 (2 regular papers) -->
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td>09:45 - 10:05</td>
                        <td>Sahand Kashani, Stuart Byma and James Larus (EPFL)
                            <br><i>IMPACT: Interval-based Multi-pass Proteomic Alignment with Constant Traceback</i></td>
                    </tr>
                    <tr>
                        <td>10:05 - 10:25</td>
                        <td>Damla Senol Cali, Jeremie S. Kim, Saugata Ghose, Can Alkan and Onur Mutlu
                            <br><a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bby017/4958758?redirectedFrom=fulltext" target="_blank"><i>Nanopore Sequencing Technology and Tools for Genome Assembly: Computational Analysis of the Current State, Bottlenecks and Future Directions</i></a></td>
                    </tr>

                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>10:25 - 11:00</td>
                        <td>Coffee break</td>
                    <tr><td><br></td><td><br></td></tr>

                        <td><strong>11:00 - 11:55</strong></td>
                        <td><strong>Invited Talk: Ananth Kalyanaraman (WSU)</strong>
                            <!--<br><i>“Next-Generation Sequencing: Big Data meets High Performance Computing Architectures”</i></strong>-->
                            </td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>


                    <!-- Paper session 2 (3 regular papers) -->
                    <tr>
                        <td>11:55 - 12:15</td>
                        <td>Xueqi Li (UCSB, KITS)
                            <br><i>Scalable In-Memory System for Genomic Analysis</i></td>

                    </tr>
                    <tr>
                        <td>12:15 - 12:35</td>
                        <td>Meysam Roodi, Zahra Lak and Andreas Moshovos (Univ. of Toronto)
                            <br><i>Accelerating GATK Variant Calling HaplotypeCaller Tool</i></td>
                    </tr>
                    <!--<tr>
                        <td>11:15 - 11:35</td>
                        <td>Regular paper session: 1 papers</td>
                    </tr>-->

                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>12:35 - 14:00</td>
                        <td>Lunch</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td><strong>14:00 - 15:00</strong></td>
                        <td><strong>Keynote 2: Onur Mutlu (ETH, CMU)</strong></td>
                            <!--<br><i>“Automata Processor and its Applications in Bioinformatics”</i></strong></td>-->
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td>15:00 - 15:20</td>
                        <td>Jeremie Kim, Damla Senol, Hongyi Xin, Donghyuk Lee, Saugata Ghose, Mohammed Alser, Hasan Hassan, Oguz Ergin, Can Alkan and Onur Mutlu
                            <br>
                            <a href="https://arxiv.org/pdf/1708.04329.pdf" target="_blank"><i>GRIM-filter: fast seed filtering in read mapping using emerging memory technologies</i></a>&nbsp;
                            <!--&nbsp;&nbsp;(<a href="https://arxiv.org/pdf/1708.04329.pdf">slides</a>)-->
                        </td>
                    </tr>
                    <tr>
                        <td>15:20 - 15:40</td>
                        <td>Roman Kaplan, Leonid Yavits and Ran Ginsar (Technion, Israel Institute of Technology)
                            <br><a href="https://arxiv.org/ftp/arxiv/papers/1809/1809.01127.pdf" target="_blank"><i>RASSA: Resistive Pre-Alignment Accelerator for Approximate DNA Long Read Mapping</i></a></td>
                    </tr>


                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>15:40 - 16:00</td>
                        <td>Coffee break</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>


                    <tr>
                        <td>16:00 - 16:20</td>
                        <td>Wenqin Huangfu, Shuangchen Li, Xing Hu and Yuan Xie (UCSB)
                            <br><a href="https://seal.ece.ucsb.edu/sites/seal.ece.ucsb.edu/files/publications/a59-huangfu.pdf" target="_blank"><i>RADAR: A 3D-ReRAM based DNA Alignment Accelerator Architecture</i></a></td>
                    </tr>
                    <tr>
                        <td>16:20 - 16:40</td>
                        <td>Zheming Jin and Hal Finkel (ANL)
                            <br><i>Exploring the Random Network of Hodgkin and Huxley Neurons with Exponential Synaptic Conductances on OpenCL FPGA Platform</i></td>
                    </tr>
                    <tr>
                        <td>16:20 - 16:40</td>
                        <td>Meysam Roodi, Zahra Lak and Andreas Moshovos (Univ. of Toronto)
                            <br><i>Improving BWA-MEM performance</i></td>
                    </tr>
                    <tr>
                        <td>16:50 - 17:00</td>
                        <td>Closing remarks</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>
                    <!-- Paper session 3 (3 regular papers) -->
                    <!--<tr>
                        <td>14:10 - 14:30</td>
                        <td>Tommy Tracy Ii, Jack Wadden, Kevin Skadron and Mircea Stan (UVA)
                            <br><i>Streaming Gap-Aware Seed Alignment on the Cache Automaton</i></td>
                    </tr>
                    <tr>
                        <td>14:30 - 14:50</td>
                        <td>Roman Kaplan, Leonid Yavits and Ran Ginosar (Technion)
                            <br><i>Processing-in-Storage Architecture for Large-Scale Biological Sequence Alignment</i></td>
                    </tr>
                    <tr>
                        <td>14:50 - 15:10</td>
                        <td>Xueqi Li, Guangming Tan, Yuanrong Wang and Ninghui Sun (ICT)
                            <br><i>The Genomic Benchmark Suite: Characterization and Architecture Implications</i></td>
                    </tr>-->

                    <!--<tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>15:10 - 15:30</td>
                        <td>Coffee break</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>
                    -->

                    <!--<tr>
                        <td>15:30 - 16:10</td>
                        <td><strong>Invited Talk: Can Alkan (Bilkent University)
                            <br><i>"Addressing Computational Burden to Realize Precision Medicine"</i></strong>&nbsp;
                            (<a target="_blank" href="./slides/2018/alkan-aacbb.pptx">slides</a>)
                        </td>
                    </tr>-->

                    <!-- Paper session 4 (2 regular papers) -->
                    <!--<tr>
                        <td>16:10 - 16:30</td>
                        <td>Sergiu Mosanu and Mircea Stan (UVA)
                            <br><i>Burrows-Wheeler Short Read Aligner on AWS EC2 F1</i>&nbsp;
                            (<a target="_blank" href="./slides/2018/aacbb-serg-presi.pdf">slides</a>)
                        </td>
                    </tr>
                    <tr>
                        <td>16:30 - 16:50</td>
                        <td>Angélica Alejandra Serrano-Rubio, Amilcar Meneses-Viveros, Guillermo B. Morales-Luna and Mireya Paredes-López (CINVESTAV-IPN)
                            <br><i>Towards BIMAX: Binary Inclusion-MAXimal parallel implementation for gene expression analysis</i></td>
                    </tr>


                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>16:50 - 17:00</td>
                        <td>Short break</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>-->

                    <!-- Paper session 3 (3 regular papers) -->
                    <!--<tr>
                        <td>17:00 - 17:15</td>
                        <td>Meysam Taassori+, Anirban Nag+, Keeton Hodgson+, Ali Shafiee* and Rajeev Balasubramonian+ (+Univ. of Utah, *Samsung Electronics)
                            <br><i>Memory: The Dominant Bottleneck in Genomic Workloads</i>&nbsp;
                            (<a target="_blank" href="./slides/2018/HPCA_2018_Genomics_An2_mychange.pdf">slides</a>)
                        </td>
                    </tr>
                    <tr>
                        <td>17:15 - 17:30</td>
                        <td>Meysam Roodi and Andreas Moshovos (Univ. of Toronto)
                            <br><i>Gene Sequencing: Where Time Goes</i></td>
                    </tr>
                    <tr>
                        <td>17:30 - 17:45</td>
                        <td>Calvin Bulla, Lluc Alvarez and Miquel Moreto (BSC)
                            <br><i>Are Next-Generation HPC Systems Ready for Population-level Genomics Data Analytics?</i>&nbsp;
                            (<a target="_blank" href="./slides/2018/20180224-aacbb-mmoreto.pdf">slides</a>)
                        </td>
                    </tr>

                    <tr><td><br></td><td><br></td></tr>
                    <tr>
                        <td>17:45 - 17:50</td>
                        <td>Closing remarks</td>
                    </tr>
                    <tr><td><br></td><td><br></td></tr>

                    <tr>
                        <td></td>
                        <td>
                            <h4>Social Event</h4>
                        </td>
                    </tr>-->

                    <!--<tr>
                        <td><strong>18:15</strong></td>
                        <td>Bus leaves to social event (Heurigen)</td>
                    </tr>-->

                </table>
            </div>

        </div>
        <div class="row">
          <div class="col-lg-12">
                <h2 class="padding-bottom-15">Keynote Talks</h2>
                <ul class="timeline">

                    <li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/omutlu.jpg" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://people.inf.ethz.ch/omutlu/">Onur Mutlu, ETH Zurich / CMU </a></h4>
                                <h3 class="keynote-subheading">“Accelerating Genome Analysis: A Primer on an Ongoing Journey”</h3>
                                <h5 class="subheading">8:45 - 9:45</h5>
                            </div>
                            <div class="timeline-body">
                                <p class="text-muted padding-bottom-15"><strong>Talk abstract: </strong>
                                    Genome analysis is the foundation of many scientific and medical discoveries as well as a key pillar of personalized medicine.
                                    Any analysis of a genome fundamentally starts with the reconstruction of the genome from its sequenced fragments.
                                    This process is called read mapping.
                                    One key goal of read mapping is to find the variations that are present between the sequenced genome and reference genome(s) and to tolerate the errors introduced by the genome sequencing process.
                                    Read mapping is currently a major bottleneck in the entire genome analysis pipeline because state-of-the-art genome sequencing technologies are able to sequence a genome much faster than the computational techniques that are employed to reconstruct the genome.
                                    New sequencing technologies, like nanopore sequencing, greatly exacerbate this problem while at the same time making genome sequencing much less costly.
                                    <br>
                                    <br>
                                    This talk describes our ongoing journey in greatly improving the
                                    performance of genome read mapping. We first provide a brief
                                    background on read mappers that can comprehensively find variations
                                    and tolerate sequencing errors. Then, we describe both algorithmic and
                                    hardware-based acceleration approaches. Algorithmic approaches exploit
                                    the structure of the genome as well as the structure of the underlying
                                    hardware. Hardware-based acceleration approaches exploit specialized
                                    microarchitectures or new execution paradigms like processing in
                                    memory. We show that significant improvements are possible with both algorithmic and hardware-based approaches and their combination.
                                    We conclude with a foreshadowing of future challenges brought about by very low cost yet highly error prone new sequencing technologies.

                                </p>

                            </div>
                        </div>
                    </li>

                    <li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/2019/william-dally_profilephoto.jpg" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://profiles.stanford.edu/william-dally">William Dally, Stanford & NVIDIA</a></h4>
                                <h3 class="keynote-subheading">Talk title: TBD</h3>
                                <h5 class="subheading">14:00-15:00</h5>
                            </div>
                            <div class="timeline-body">
                                <!--<p class="text-muted padding-bottom-15"><strong>Talk abstract: </strong>
                                    This talk will introduce the Micron Automata Processor (AP), a novel computing architecture that enables massively parallel execution of numerous non-deterministic finite automata.
                                    The processor inspires a new programming paradigm of solving problems using complex pattern matching engines executed over streaming data.
                                    The first part of this talk will focus on the processor characteristics, programming and execution environment, and design principles we discovered that are of value in developing applications on the AP.
                                    The second part will feature my group's research on developing bioinformatics algorithms for the AP including database search and motif detection.
                                </p>-->
                            </div>
                        </div>
                    </li>
                </ul>
            </div>
        </div>
        <div class="row">
            <div class="col-lg-12">
                <h2 class="padding-bottom-15 padding-top-40">Invited Talk</h2>
                <ul class="timeline">

                    <li class="timeline-inverted">
                        <div class="timeline-image">
                            <img class="rounded-circle img-fluid" src="img/speakers/2019/Ananth.jpg" alt="">
                        </div>
                        <div class="timeline-panel">
                            <div class="timeline-heading">
                                <h4><a target="_blank" href="https://www.eecs.wsu.edu/~ananth/">Ananth Kalyanaraman, Washington State University</a></h4>
                                <h3 class="keynote-subheading">Talk title: TBD</h3>
                                <h5 class="subheading">11:00-11:55</h5>
                            </div>
                            <div class="timeline-body">
                                <!--<p class="text-muted padding-bottom-15"><strong>Talk abstract: </strong>
                                    The progress of NGS has a major impact on medical and genomic research.
                                    This high-throughput technology can now produce billions of short DNA fragments in excess of a few Terabytes of data in a single run.
                                    This leads to massive datasets used by a wide range of applications including personalized cancer treatment and precision medicine.
                                    In addition to the hugely increased throughput, the cost of using high-throughput technologies has been dramatically decreasing.
                                    Low sequencing cost of around US$1K per genome has now rendered large population-scale projects feasible.
                                    However, in order to make effective use of the produced data, the design of big data algorithms and their efficient implementation on modern HPC systems is required.
                                    In this talk, I will present the design of scalable algorithms for metagenomic read classification and for massively parallel hash maps on multi-GPU nodes.

                                </p>-->
                                <!--<p class="text-muted"><strong>Short speaker bio: </strong> TBD</p>-->
                            </div>
                        </div>
                    </li>

                </ul>
            </div>
        </div>
      </div>
    </section>


    <section id="program-committee">
        <div class="container">
            <div class="row">
                <div class="col-sm-2"></div>
                <div class="col-lg-8 ">
                    <h2 class="section-heading text-center">Program Committee</h2>
                    <ul id="program-committee-list">
                        <li><a href="http://www.eecs.wsu.edu/~ananth/">Ananth Kalyanaraman, WSU</a></li>
                        <li><a href="http://www.cs.bilkent.edu.tr/~calkan/">Can Alkan, Bilkent University</a></li>
                        <li><a href="http://www.cs.rochester.edu/~ipek/Home.html">Engin Ipek, University of Rochester</a></li>
                        <li><a href="http://vast.cs.ucla.edu/people/faculty/jason-cong">Jason Cong, UCLA</a></li>
                        <li><a href="http://lph.ece.utexas.edu/merez/MattanErez/Home">Mattan Erez, UT Austin</a></li>
                        <li><a href="http://www.ece.virginia.edu/faculty/stan.html">Mircea Stan, UVA</a></li>
                        <li><a href="https://people.inf.ethz.ch/omutlu/">Onur Mutlu, ETH/CMU</a></li>
                        <li><a href="http://webee.technion.ac.il/~ran/">Ran Ginosar, Technion</a></li>
                        <li><a href="http://www.ece.ucsb.edu/~yuanxie/">Yuan Xie, UCSB</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </section>


    <!-- Organizers -->
    <section class="bg-light" id="organizers">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Workshop Organizers</h2>
            <h3 class="padding-bottom-15 text-muted">From Technion, Israel Institute of Technology</h3>
          </div>
        </div>
        <div class="row">
            <!--<div class="col-sm-1"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Leonid.jpg" alt="">
              <h4>Leonid Yavits</h4>
              <p class="oranizer-email">leonid.yavits@nububbles.com</p>
                <p class="text-left"><strong>Short bio:</strong> Leonid received his MSc and PhD in Electrical Engineering from the Technion. After graduating, he co-founded VisionTech where he co-designed a single chip MPEG2 codec. Following VisionTech’s acquisition by Broadcom, he co-founded Horizon Semiconductors where he co-designed a Set Top Box on chip for cable and satellite TV. <br>
                    Leonid is a postdoc fellow in Electrical Engineering in the Technion. He co-authored a number of patents and research papers on SoC and ASIC. His research interests include non von Neumann computer architectures and processing in memory</p>
            </div>
          </div>
            <!--<div class="col-sm-2"></div>-->
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Roman.png" alt="">
              <h4>Roman Kaplan</h4>
              <p class="oranizer-email">romankap@gmail.com</p>
                <p class="text-left"><strong>Short bio:</strong>
                    Roman received his BSC and MSc from the faculty of Electrical Engineering, Technion, Israel in 2009 and 2015, respectively. He is now a PhD candidate in the same faculty under the supervision of Prof. Ran Ginosar. <br>
                    Roman's research interests are parallel computer architectures, in-data processing, accelerators for machine learning and big data, and novel computer architectures for bioinformatics applications.
                </p>

            </div>
          </div>
        </div>
        <!--<div class="row">
          <div class="col-lg-8 mx-auto text-center">
            <p class="large text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde.</p>
          </div>
        </div>-->
      </div>
    </section>

    <!-- Contact -->
    <!--
    <section id="contact">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Contact Us</h2>
            <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
          </div>
        </div>
        <div class="row">
          <div class="col-lg-12">
            <form id="contactForm" name="sentMessage" novalidate>
              <div class="row">
                <div class="col-md-6">
                  <div class="form-group">
                    <input class="form-control" id="name" type="text" placeholder="Your Name *" required data-validation-required-message="Please enter your name.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="email" type="email" placeholder="Your Email *" required data-validation-required-message="Please enter your email address.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="phone" type="tel" placeholder="Your Phone *" required data-validation-required-message="Please enter your phone number.">
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="col-md-6">
                  <div class="form-group">
                    <textarea class="form-control" id="message" placeholder="Your Message *" required data-validation-required-message="Please enter a message."></textarea>
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="clearfix"></div>
                <div class="col-lg-12 text-center">
                  <div id="success"></div>
                  <button id="sendMessageButton" class="btn btn-xl" type="submit">Send Message</button>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    </section>
    -->

    <!-- Footer -->
    <footer>
      <div class="container">
        <div class="row">
          <div class="col-md-4">
            <span class="copyright">Copyright &copy; AACBB 2018</span>
          </div>

        </div>
      </div>
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/popper/popper.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

    <!-- Contact form JavaScript -->
    <script src="js/jqBootstrapValidation.js"></script>
    <script src="js/contact_me.js"></script>

    <!-- Custom scripts for this template -->
    <script src="js/agency.js"></script>

  </body>

</html>
