<!DOCTYPE html>
<html lang="en">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="">
  <link rel="icon" type="image/png" sizes="96x96" href="favicon.png">

  <title>AACBB - Workshop on Accelerator Architecture for Computational Biology and Bioinformatics</title>

  <!-- Bootstrap core CSS -->
  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom fonts for this template -->
  <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
  <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet'
    type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

  <!-- Custom styles for this template -->
  <link href="css/agency.css" rel="stylesheet">

</head>

<body id="page-top">

  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
    <div class="container">
      <a class="navbar-brand js-scroll-trigger" href="#page-top">Back to top</a>
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse"
        data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false"
        aria-label="Toggle navigation">
        Menu
        <i class="fa fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbarResponsive">
        <ul class="navbar-nav ml-auto">
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#about">About</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#cfp">Call for Participation</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#program-committee">Program Committee</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
          </li>
          <li class="nav-item">
            <!-- <a class="nav-link" href="AACBB-2019.html" target="_blank">AACBB'19</a> -->
          </li>
          <li class="nav-item">
            <a class="nav-link" href="AACBB-2022.html" target="_blank">AACBB'22</a>
          </li>

          <!-- 	<li class="nav-item">
              <a class="nav-link" href="AACBB-2018.html" target="_blank">AACBB'18</a>
            </li> -->

        </ul>
      </div>
    </div>
  </nav>

  <!-- Header -->
  <header class="masthead">
    <div class="container">
      <div class="intro-text">
        <div class="intro-lead-in">The 5<sup>th</sup> Workshop on</div>
        <div class="intro-heading">Accelerator Architecture in <br> Computational Biology and Bioinformatics</div>

      </div>
      <h3 class="heading-subtitle">June 18<sup>th</sup>, 2023</h3>
      <h3 class="heading-subtitle">In conjunction with 50th IEEE International Symposium on Computer Architecture</h3>
      <h3 class="heading-subtitle">Orlando, Florida, USA</h3>
      <h3 class="section-heading text-center"><a href="https://aacbb-workshop.github.io/">Workshop website</a></h3>
      <h3 class="section-heading text-center"><a href="https://easychair.org/conferences/?conf=aacbb2023">Submission
          link</a></h3>
      <h6>Submission Deadline - April 20, 2023, EoD AoE</h6>
      <h6>Notifications - April 30, 2023</h6>
    </div>
  </header>


  <section id="about">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h2 class="section-heading">About</h2>
          <p class="about-text">Over the last decade, the advent of high-throughput sequencing techniques brought an
            exponential growth
            in sequenced data. At the same time, the single-thread performance continued to improve by only a few
            percent point annually.
            The growing gap between the performance demand to performance supply became a significant challenge in the
            path to scientific discovery.
            The computational bottleneck of genome analysis pipelines became even more apparent during the Covid-19
            pandemic,
            where fast and accurate virus detection and classification tools have been critical for the worldwide
            genomic surveillance system.
            <br>

            <br>The gap between the performance of a conventional computer architecture and the biological data
            processing requirements is growing.
            For example, assembling a human genome from 3rd generation sequenced data may require hundreds of CPU hours.
            Hence, computational biology and bioinformatics will have to rely on hardware accelerators to allow
            processing to keep up with
            the exploding amount of sequenced data.<br>

            <br>In a typical application, the dominant portion of the runtime is spent in a small number of
            computational kernels,
            making it an excellent target for hardware acceleration. The combination of increasingly large datasets and
            high-performance
            computing requirements make computational biology a prime candidate to benefit from accelerator architecture
            research.
            Potential directions include heterogeneous architectures, 3D integration, near-data processing, in-data
            processing,
            and reconfigurable architectures.<br>

            <br>This workshop will focus on architecture and design of hardware accelerators for computational biology
            and bioinformatics problems.
            We plan to present and discuss a variety of acceleration techniques, accelerator architectures and their
            implications on the development
            of computational biology. This year, we plan to extend the industry angle, by providing a keynote and
            invited talks from leading
            industry research specialists.<br>
          </p>
        </div>
      </div>
    </div>
  </section>


  <section id="cfp">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 ">
          <h2 class="section-heading text-center">List of Topics</h2>
          <h5>
            This workshop focuses on architecture and design of hardware and software accelerators for computational
            biology and bioinformatics problems.
            <br><br>Topics of interest include, but are not limited to the following:
          </h5>
          <ul>
            <li>Hardware and software algorithms/applications in the fields of computational biology, such as (but not
              limited to)</li>
            <ul style="padding-left:20px">
              <li>Bioinformatics</li>
              <li>Genomics</li>
              <li>Proteomics</li>
              <li>Protein structure prediction</li>
              <li>Covid-19 pandemic</li>
              <li>Antimicrobial resistance</li>
            </ul>
            <li>Bioinformatics and computational biology accelerator architecture and design based on (but not limited
              to):</li>
            <ul style="padding-left:20px">
              <li>Heterogeneous architectures</li>
              <li>3D memory-logic stack </li>
              <li>Near-memory and in-memory processing</li>
              <li>FPGAs and reconfigurable</li>
            </ul>
            <li>Emerging memory technologies and their impact on bioinformatics and computational biology</li>
            <li>Impact of bioinformatics and biology applications on computer architecture research</li>
            <li>Bioinformatics and computational biology-inspired hardware/software trade-offs</li>
          </ul>
          <!-- <h2 class="section-heading text-center">Registration information</h2>
                    <h6><br><a href = "https://www.hpca-conf.org/2020/registration/">Workshop registration (minimum requirement is 1-day workshop registration)</a></h6>
					<h6><br><a href = "https://www.hpca-conf.org/2020/venue-hotel/">HPCA Venue and hotel information</a></h6>
					<h6><br>Workshop will be held in room Capri</h6> -->

          <!--  <h5 class="padding-bottom-10">Interested authors are encouraged to submit papers (2-5 pages) through 
                        <a href="https://easychair.org/conferences/?conf=aacbb2020">EasyChair</a>
					<ul>
                        <li><h6>Submitting and presenting a paper in the workshop does not preclude publication in other venues.</h6></li>
                    </ul>
                    </h5> -->
          <!-- <h6>For a PDF version of the CFP click here: <a href="CFP/Workshop%20on%20AACBB%20-%202019%20-%20extended%20CFP.pdf" target="_blank">Download CFP</a></h6>
                    <p>All papers will be evaluated based on their relation to the subject, novelty, fundamental insights, experimental evaluation, and potential for long-term impact.</p> -->
          <!--                     <p>The full version should be a PDF file in HPCA26 submission format. For formatting instructions please refer to:
                        <a href="https://www.hpca-conf.org/2020/hpca-2020-guidelines-for-submissions/">HPCA Guidelines for Submission</a>
                    </p>

                     <h2>Important Dates</h2>
                    <h5><u>Submission deadline</u>: <del>Dec 15th, 2019 </del><ins><b><font size="6">Dec 30th, 2019</font></b></ins></h5>
					 -->

        </div>
      </div>
    </div>
  </section>

  <!-- About -->

  <section id="schedule">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h2 class="section-heading">Schedule</h2>
        </div>
      </div>
      <div class="row padding-bottom-20">
        <div class="col-lg-1"></div>
        <div class="col-lg-10">
          <table id="schedule-table" class="table-striped table-responsive table-hover">
            <tr>
              <th style="width:120px"></th>
              <th></th>
            </tr>

            <tr>
              <td><br></td>
              <td><br></td>
            </tr>
            <tr>
              <td>8:30 - 8:40</td>
              <td>Opening Remarks</td>
            </tr>
            <tr>
              <td><br></td>
              <td><br></td>
            </tr>

            <tr>
              <td><strong>8:40 - 9:30</strong></td>
              <td><strong>Katherine A. Yelick</strong><br>
                <i>"Genomic Analysis at Scale: Mapping Irregular Computations to Advanced Architectures"</i>
              </td>
            </tr>
            <tr>
              <td>9:30 - 9:55</td>
              <td>Naomie Abecassis, Juan Gomez Luna, Ran Ginosar, Aphélie Moisson-Franckhauser and Leonid Yavits<br>
                <i>"GAPiM: a hardware acceleration of Genome Analysis pipeline using Processing in Memory"</i>
              </td>
            </tr>
            <tr>
              <td>9:55-10:20</td>
              <td>Courtney Golden, Dan Ilan, Nicholas Cebry and Christopher Batten<br>
                <i>"Accelerating Seed Location Filtering in DNA Read Mapping Using a Commercial Compute-in-SRAM
                  Architecture"</i>
              </td>
            </tr>
            <tr>
              <td>10:20-10:40</td>
              <td>Coffee break</td>
            </tr>

            <tr>
              <td><br></td>
              <td><br></td>
            </tr>

            <td>10:40 - 11:05</td>
            <td>Alon Rashelbach, Ori Rottenstreich and Mark Silberstein
              <br><i>"Nucleotide String Indexing using Range Matching "</i>
            </td>
            </tr>

            <tr>
              <td>11:05 - 11:30</td>
              <td>Meven Mognol, Julien Legriel and Dominique Lavenier
                <br><i>"Parallelization of the banded Needleman & Wunsch algorithm on UPMEM PiM architecture for long
                  DNA sequence alignment"</i>
              </td>
            </tr>

            <tr>
              <td>11:30 - 11:55</td>
              <td>Yilin Feng, Zheyu Li, Gulsum Gudukbay Akbulut, Vijaykrishnan Narayanan, Mahmut Taylan Kandemir and
                Chitaranjan Das
                <br><i>"FPGA Implementation of Adaptive Band Event Alignment using High-Level Synthesis "</i>
              </td>
            </tr>

            <tr>
              <td><br></td>
              <td><br></td>
            </tr>

            <tr>
              <td>11:55 - 13:00</td>
              <td>Lunch</td>
            </tr>
            <tr>
              <td><br></td>
              <td><br></td>
            </tr>

            <tr>
              <td><strong>13:00 - 13:50</strong></td>
              <td><strong>Dominique Lavenier</strong>
                <br><i>"Exploring genomic algorithms on UPMEM Processing-in-Memory Architecture" </i>
              </td>
            </tr>

            <tr>
              <td>13:50 - 14:15</td>
              <td>Joël Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez-Luna, Nika Mansouri Ghiasi and Onur
                Mutlu<br>
                <i>"Scrooge: A Fast and Memory-Frugal Genomic Sequence Aligner for CPUs, GPUs, and ASICs "</i>
              </td>
            </tr>

            <tr>
              <td>14:15 - 14:40</td>
              <td>Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad
                Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu
                <br>
                <i>"TargetCall: Eliminating the Wasted Computation in Basecalling via Pre-Basecalling Filtering "</i>
              </td>
            </tr>

            <tr>
              <td><br></td>
              <td><br></td>
            </tr>

            <tr>
              <td>14:40 - 15:00</td>
              <td>Coffee break</td>
            </tr>

            <tr>
              <td><br></td>
              <td><br></td>
            </tr>


            <tr>
              <td><strong>15:00 - 15:50</strong></td>
              <td><strong>Damla Senol Cali</strong>
                <br><i>"Accelerating the Transformation in How We Analyze the Human Genome"</i></a>
              </td>
            </tr>

            <tr>
              <td>15:50 - 16:15</td>
              <td>Can Firtina, Nika Mansouri Ghiasi, Joel Lindegger, Gagandeep Singh, Meryem Banu Cavlak, Haiyu Mao and
                Onur Mutlu
                <br><i>"RawHash: Enabling Fast and Accurate Real-Time Analysis of Raw Nanopore Signals for Large
                  Genomes"</i></a>
              </td>
            </tr>

            <tr>
              <td>16:15 - 16:40</td>
              <td>Gagandeep Singh, Mohammed Alser, Alireza Khodamoradi, Kristof Denolf, Can Firtina, Meryem Banu Cavlak,
                Henk Corporaal and Onur Mutlu
                <br><i>"RUBICON: A Framework for Designing Efficient Deep Learning-Based Genomic Basecallers "</i></a>
              </td>
            </tr>

            <tr>
              <td>16:40 - 17:05</td>
              <td>Can Firtina, Jisung Park, Mohammed Alser, Jeremie S. Kim, Damla Senol Cali, Taha Shahroodi, Nika
                Mansouri Ghiasi, Gagandeep Singh, Konstantinos Kanellopoulos, Can Alkan and Onur Mutlu
                <br><i>"BLEND: a fast, memory-efficient and accurate mechanism to find fuzzy seed matches in genome
                  analysis"</i></a>
              </td>
            </tr>

            <tr>
              <td>17:05 - 17:10</td>
              <td>Closing remarks</td>
            </tr>
            <tr>
              <th style="width:120px"></th>
              <th></th>
            </tr>


          </table>
        </div>

      </div>


      <div class="container">
        <div class="row">
          <div class="col-sm-2"></div>
          <div class="col-lg-8 ">
            <h2 class="section-heading text-center">Keynote Speakers</h2>
            <!--                     <ul id="program-committee-list">
                        <li><a href="https://www.soe.ucsc.edu/people/benedict">Benedict Paten, UC Santa Cruz Genomics Institute</a></li>
                         <li><a href="https://people.epfl.ch/james.larus?lang=en">James Larus, EPFL</a></li>
                         <li><a href="https://www.deshawresearch.com/people_hdwdes_gagliardo.html">Joe Gagliardo, D. E. Shaw Research</a></li>
                     
					</ul> -->
          </div>
        </div>
      </div>


      <div class="row">
        <div class="col-lg-12">
          <!-- <h2 class="section-heading text-center">Talks</h2> -->
          <ul class="timeline">

            <li class="timeline-inverted">
              <div class="timeline-image">
                <img class="rounded-circle img-fluid" src="img/speakers/2023/Kathy.png" alt="">
              </div>
              <div class="timeline-panel">
                <div class="timeline-heading">
                  <h4><a target="_blank" href="https://vcresearch.berkeley.edu/faculty/katherine-yelick">Katherine A.
                      Yelick</a></h4>
                  <h3 class="keynote-subheading">Professor of Electrical Engineering and Computer Science , University
                    of California, Berkeley</h3>
                  <p class="text-muted"><strong>Genomic Analysis at Scale: Mapping Irregular Computations to Advanced
                      Architectures</strong></p>
                </div>
                <div class="timeline-body">
                  <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
                    Katherine Yelick is the Vice Chancellor for Research at the University of California, Berkeley,
                    where she also holds the Robert S. Pepper Distinguished Professor of Electrical Engineering and
                    Computer Sciences. She is also a Senior Faculty Scientist at Lawrence Berkeley National Laboratory.
                    Her research is in high performance computing, programming systems, parallel algorithms, and
                    computational genomics and she currently leads the ExaBiome project on Exascale Solutions for
                    Microbiome Analysis. Yelick was Director of the National Energy Research Scientific Computing Center
                    (NERSC) from 2008 to 2012 and led the Computing Sciences Area at Lawrence Berkeley National
                    Laboratory from 2010 through 2019, where she oversaw NERSC, the Energy Sciences Network (ESnet) and
                    the Computational Research Division. She earned her Ph.D. in Electrical Engineering and Computer
                    Science from MIT and is a member of the National Academy of Engineering and the American Academy of
                    Arts and Sciences. She is a Fellow of the Association for Computing Machinery (ACM) and the American
                    Association for the Advancement of Sciences (AAAS), and she is a recipient of the ACM/IEEE Ken
                    Kennedy award and the ACM-W Athena award. Yelick was an Associate Dean in the Division of Computing,
                    Data Science, and Society from 2020-2021 and Vice Chancellor for Research at UC Berkeley starting in
                    January 2022.
                  </p>
                </div>
              </div>
            </li>


            <li class="timeline-inverted">
              <div class="timeline-image">
                <img class="rounded-circle img-fluid" src="img/speakers/2023/PhotoDLavenier.jpg" alt="">
              </div>
              <div class="timeline-panel">
                <div class="timeline-heading">
                  <h4><a target="_blank" href="https://lavenier.net/homepage/">Dominique Lavenier</a></h4>
                  <h3 class="keynote-subheading">CNRS research Director, IRISA/INRIA</h3>
                  <p class="text-muted"><strong>Exploring genomic algorithms on UPMEM Processing-in-Memory
                      Architecture</strong></p>
                </div>
                <div class="timeline-body">
                  <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
                    Dominique Lavenier is a CNRS research fellow and member of the GenScale bioinformatics team at
                    IRISA/INRIA, Rennes, which he created in 2012. His current research interests include
                    bioinformatics, data structures, genomics, parallelism, algorithm optimization, in-memory processing
                    and DNA storage. In the past, he has designed several hardware prototypes for molecular biology,
                    bioinformatics or genomic applications, including VLSI chips and FPGA + Memory architectures. Since
                    2015, he has been collaborating continuously with UPMEM to investigate different ways to parallelize
                    genomic algorithms on in-memory processing systems.
                  </p>
                </div>
              </div>
            </li>

            <li class="timeline-inverted">
              <div class="timeline-image">
                <img class="rounded-circle img-fluid" src="img/speakers/2023/Damla.jpg" alt="">
              </div>
              <div class="timeline-panel">
                <div class="timeline-heading">
                  <h4><a target="_blank" href="https://damlasenolcali.github.io/">Damla Senol Cali</a></h4>
                  <h3 class="keynote-subheading">Hardware Acceleration specialist, Bionano Genomics</h3>
                  <p class="text-muted"><strong>Accelerating the Transformation in How We Analyze the Human
                      Genome</strong></p>
                </div>
                <div class="timeline-body">
                  <p class="text-muted padding-bottom-15"><strong>Short Bio: </strong>
                    Dr. Damla Senol Cali is a Staff Software Engineer, focusing on Hardware Acceleration at Bionano.
                    Her work at Bionano includes leading efforts to develop a new hardware acceleration platform for the
                    analysis of Bionano’s
                    Optical Genome Mapping (OGM) data and collaborating with bioinformatics scientists and software
                    engineering teams to create
                    powerful frameworks to accelerate bioinformatics workloads and emerging applications. She received
                    her Ph.D.
                    degree in Computer Engineering from Carnegie Mellon University in 2021.
                    Her work broadly spans bioinformatics and computer architecture, focusing on hardware/software
                    co-design for accelerating bioinformatics
                    applications and genomic data analysis. During her Ph.D.,
                    she has published several papers that appeared in top computer architecture venues including MICRO
                    and ISCA.
                    She obtained her M.S. in Computer Engineering from Carnegie Mellon University in 2019,
                    and her B.S. in Computer Engineering from Bilkent University, Turkey in 2015. During her Ph.D., she
                    also interned at Intel Labs in 2018 and 2020.
                  </p>
                </div>
              </div>
            </li>
          </ul>
        </div>
      </div>



  </section>




  <section id="program-committee">
    <div class="container">
      <div class="row">
        <div class="col-sm-2"></div>
        <div class="col-lg-8 ">
          <h2 class="section-heading text-center">Program Committee</h2>
          <ul id="program-committee-list">
            <li><a href="http://www.eecs.wsu.edu/~ananth/">Ananth Kalyanaraman, WSU</a></li>
            <li><a href="http://www.cs.bilkent.edu.tr/~calkan/">Can Alkan, Bilkent University</a></li>
            <li><a href="http://www.cs.rochester.edu/~ipek/Home.html">Engin Ipek, University of Rochester</a></li>
            <li><a href="http://vast.cs.ucla.edu/people/faculty/jason-cong">Jason Cong, UCLA</a></li>
            <li><a href="http://lph.ece.utexas.edu/merez/MattanErez/Home">Mattan Erez, UT Austin</a></li>
            <li><a href="http://www.ece.virginia.edu/faculty/stan.html">Mircea Stan, UVA</a></li>
            <li><a href="https://people.inf.ethz.ch/omutlu/">Onur Mutlu, ETH/CMU</a></li>
            <li><a href="http://webee.technion.ac.il/~ran/">Ran Ginosar, Technion</a></li>
            <li><a href="https://asic2.group/team/current-members/?stl=2066">Ronnie Ronen, Technion</a></li>
            <li><a href="http://www.ece.ucsb.edu/~yuanxie/">Yuan Xie, UCSB</a></li>
            <li><a href="https://damlasenolcali.github.io/">Damla Senol Cali, Bionano Genomics</a></li>
          </ul>
        </div>
      </div>
    </div>
  </section>




  <!-- Organizers -->
  <section class="bg-light" id="organizers">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h2 class="section-heading">Workshop Organizers</h2>
        </div>
      </div>
      <div class="row">
        <!--<div class="col-sm-1"></div>-->
        <div class="col-sm-6">
          <div class="team-member">
            <img class="mx-auto rounded-circle" src="img/team/Leonid.jpg" alt="">
            <h4>Leonid Yavits</h4>
            <p class="oranizer-email">Bar Ilan University</p>
            <p class="text-left"><strong>Short bio:</strong> Leonid Yavits is with Bar Ilan University, Israel. He
              received his MSc and PhD in Electrical Engineering from the Technion. After graduating, he co-founded
              VisionTech where he co-designed the world’s first single chip MPEG2 codec. Following VisionTech’s
              acquisition by Broadcom, he managed Broadcom Israel R&amp;D and co-developed a number of video compression
              products. Later Leonid co-founded Horizon Semiconductors where he co-designed a Set Top Box-on-chip for
              cable and satellite TV. Leonid’s research interests include non von Neumann computer architectures,
              processing in memory, and domain specific accelerators. Leonid's research work has earned several awards;
              among them: IEEE Computer Architecture Letter Journal Best Paper Awards for 2015 and 2017 and best poster
              award at ICS High Performance Conference 2017.</p>
          </div>
        </div>
        <!--<div class="col-sm-2"></div>-->
        <div class="col-sm-6">
          <div class="team-member">
            <img class="mx-auto rounded-circle" src="img/team/Yatish.png" alt="">
            <h4>Yatish Turakhia</h4>
            <p class="oranizer-email">University of California, San Diego</p>
            <p class="text-left"><strong>Short bio:</strong> Yatish Turakhia is assistant professor of Electrical and
              Computer Engineering (ECE) at UCSD. He was previously a postdoctoral scholar at the Genomics Institute at
              UC Santa Cruz and received his PhD student in from Stanford University. His work has been conferred with
              the best paper award at ASPLOS, IEEE Micro Top picks and NVIDIA graduate fellowship.</p>
          </div>
        </div>
      </div>
    </div>

    <!-- <div class="row">
          <div class="col-lg-12 text-center">
            <h3 class="padding-bottom-15 text-muted">From UCSD</h3>
          </div>
        </div>
        
        
        <div class="row">
            <div class="col-sm-1"></div>
          <div class="col-sm-6">
            <div class="team-member">
              <img class="mx-auto rounded-circle" src="img/team/Saurabh.jpg" alt="">
              <h4>Saurabh Shetty</h4>
              <p class="oranizer-email">sshetty@ucsd.edu</p>
                <p class="text-center"><strong>Short bio:</strong> Saurabh is a first year Masters student in Computer Science Engineering at University of California San Diego.
                He is interested in the hardware-acceleration of genome sequence alignment and applications of heterogenous computing to bioinformatic algorithms</p>
            </div>
          </div>
        </div> -->

    <!--<div class="row">
          <div class="col-lg-8 mx-auto text-center">
            <p class="large text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde.</p>
          </div>
        </div>-->
    </div>
  </section>

  <!-- Contact -->
  <!--
    <section id="contact">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Contact Us</h2>
            <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
          </div>
        </div>
        <div class="row">
          <div class="col-lg-12">
            <form id="contactForm" name="sentMessage" novalidate>
              <div class="row">
                <div class="col-md-6">
                  <div class="form-group">
                    <input class="form-control" id="name" type="text" placeholder="Your Name *" required data-validation-required-message="Please enter your name.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="email" type="email" placeholder="Your Email *" required data-validation-required-message="Please enter your email address.">
                    <p class="help-block text-danger"></p>
                  </div>
                  <div class="form-group">
                    <input class="form-control" id="phone" type="tel" placeholder="Your Phone *" required data-validation-required-message="Please enter your phone number.">
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="col-md-6">
                  <div class="form-group">
                    <textarea class="form-control" id="message" placeholder="Your Message *" required data-validation-required-message="Please enter a message."></textarea>
                    <p class="help-block text-danger"></p>
                  </div>
                </div>
                <div class="clearfix"></div>
                <div class="col-lg-12 text-center">
                  <div id="success"></div>
                  <button id="sendMessageButton" class="btn btn-xl" type="submit">Send Message</button>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    </section>
    -->

  <!-- Footer -->
  <footer>
    <div class="container">
      <div class="row">
        <div class="col-md-4">
          <span class="copyright">Copyright &copy; AACBB 2020</span>
        </div>

      </div>
    </div>
  </footer>

  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/popper/popper.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

  <!-- Contact form JavaScript -->
  <script src="js/jqBootstrapValidation.js"></script>
  <script src="js/contact_me.js"></script>

  <!-- Custom scripts for this template -->
  <script src="js/agency.js"></script>

</body>

</html>